### PR TITLE
Add 'title' command.

### PR DESCRIPTION
**Pull request by tohojo**
I found myself wanting a quick way to add simple titles to lots of images. Rather than writing a separate script I thought that just adding a new command to sigal itself might be easier. Do you think this is worth including in upstream sigal? In that case I can add test cases and documentation for it, but wanted to see if you'd want to take it at all first...

### CODE REVIEW

**Comment by saimn** [comment]
It can be useful so +1 but maybe a bit more generic, to allow writing several key/value pairs at once ? Then we also need a more generic name ;)

File: sigal/__init__.py [hunk 1]
```diff
@@ -223,3 +223,23 @@ def serve(destination, port, config):
         httpd.serve_forever()
     except KeyboardInterrupt:
         print('\nAll done!')
+
+@main.command()
+@argument('image')
+@argument('title')
+@option('-f', '--force', default=False, is_flag=True,
```
Comment (saimn) [inline sigal/__init__.py]: I would prefer `overwrite` to `force`, more explicit on what it does.

**Comment by saimn** [comment]
It could be useful for albums as well.

**Comment by tohojo** [comment]
Something like this?

File: sigal/__init__.py [hunk 2]
```diff
@@ -223,3 +223,37 @@ def serve(destination, port, config):
...
     except KeyboardInterrupt:
         print('\nAll done!')
+
+@main.command()
+@argument('target')
+@argument('keys', nargs=-1)
+@option('-o', '--overwrite', default=False, is_flag=True,
+        help='Overwrite existing .md file')
+def set_meta(target, keys, overwrite=False):
+    """Write image title to corresponding .md file.
```
Comment (saimn) [inline sigal/__init__.py]: Docstring needs to be updated, mention that target can be a directory or a media file.

File: sigal/__init__.py [hunk 3]
```diff
@@ -223,3 +223,37 @@ def serve(destination, port, config):
...
+        descfile = os.path.splitext(target)[0] + '.md'
+    if os.path.exists(descfile) and not overwrite:
+        sys.stderr.write("Description file '{}' already exists. "
+                         "Use --overwrite to overwrite it.\n".format(descfile))
+        sys.exit(2)
+
+    with open(descfile, "w") as fp:
+        for i in range(len(keys)//2):
+            k,v = keys[i*2:(i+1)*2]
+            fp.write("{}: {}\n".format(k.capitalize(), v))
```
Comment (saimn) [inline sigal/__init__.py]: Maybe a print with the name of the file that has been written can be useful for users ?

**Comment by saimn** [comment]
Much better, thanks :)
Could you also add a test (in `test_cli.py`) ?

**Comment by tohojo** [comment]
Yup, can do; just wanted to check with you that the interface is acceptable ;)

**Comment by tohojo** [comment]
And there's the test :)

**Comment by saimn** [comment]
:+1:

**Pull request merged**